collateral/deployment/k8s-without-helm-reference.md

```diff
@@ -0,0 +1,53 @@
+# Deploying Sourcegraph to Kubernetes (k8s) without Helm
+
+## Introduction
+This material is intended to be used as a handout to customers working with the Customer Engineering team. The materials are meant to serve as an overview and resource to aid in deep diving into Sourcegraph's documentation and common deployment tasks.
+
+**Table of Contents:**
+
+- [Introduction](#introduction)
+- [Sourcegraph Setup and Deployment](#sourcegraph-setup-and-deployment)
+	- [Prerequisites](#prerequisites)
+  - [Create your cluster](#create-your-cluster)
+  - [Configure your manifest file](#configure-your-manifest-file)
+  - [Install Sourcegraph](#install-sourcegraph)
+- [Next Steps](#next-steps)
+
+## Sourcegraph Setup and Deployment
+
+### Prerequisites
+- Ensure you have your Sourcegraph Enterprise License
```

Comment:
Up to you on whether you want to leave this, but this isn't required to deploy, and suggesting that it is confuses a lot of people historically (typically they receive the license after the instance is stood up.)

```suggestion
```

Reply:
@emchap are they modifying their manifest and applying again afterward? I guess if I was going to do this, I'd create my cluster and deploy with all my configs in the manifest file then later changes/tweaks to my manifest could be applied later if needed, but a license to get sourcegraph started seemed like a "first deploy" thing versus a later modification.

Totally open to your thoughts since you've gone through implementation here and have the insight.

Reply:
@IAmHughes The license is added in-app and is unrelated to the deployment; it's usually given to them once the deployment is stood up, and the CE will run through checks with them at this point.

Reply:
Interesting. @emchap would you say the best practice is to deploy my instance....then apply the manifest again with the license key later? As a best practice, it feels like having my license key initially when I modify my manifest for storage classes, ingress, etc makes more sense than applying it afterward.

I know you're speaking to how it has been, but is that really a best practice in general for standing up software like this? I haven't seen it that way in my past experience, but my past experience is a narrow sliver of the entire software ecosystem of course. Is there a benefit to adding the license later?